manifests/v/VivaldiTechnologies/Vivaldi/Snapshot/5.1.2514.11/VivaldiTechnologies.Vivaldi.Snapshot.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS1.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: VivaldiTechnologies.Vivaldi.Snapshot
+PackageVersion: 5.1.2514.11
+PackageLocale: en-US
+Publisher: Vivaldi Technologies AS
```

```suggestion
Publisher: Vivaldi Technologies AS.
```
to match Control Panel.